geometry/latlon.hpp

```diff
@@ -1,5 +1,7 @@
 #pragma once
 
+#include "geometry/latlon.hpp"
```

Comment:
I'm sorry that I've put [Don't Merge] comment not in title of PR.  Also, please do review of core code in the separate PR - https://github.com/mapsme/omim/pull/4451